cranelift-codegen/meta/src/cdsl/typevar.rs

```diff
@@ -406,6 +410,7 @@ pub struct TypeSet {
     pub ints: NumSet,
     pub floats: NumSet,
     pub bools: NumSet,
+    pub references: NumSet,
```

Comment:
(light suggestion, feel free to ignore if you think it's best at the moment): I think it would be nice to use "refs" instead of "references", the same way we use "ints" instead of "integers".